webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/RestTestSetUp.scala

```diff
@@ -629,6 +633,11 @@ class RestTestSetUp {
       override def startManualUpdate: Unit = ()
     }
   }
+  val techniqueCompilationStatusService: TechniqueCompilationStatusSyncService = new TechniqueCompilationStatusSyncService {
+    def syncOne(result:           EditorTechniqueCompilationResult):                      IOResult[Unit] = ().succeed
+    def unsyncOne(id:             (BundleName, Version)):                                 IOResult[Unit] = ().succeed
+    def getUpdateAndSync(results: Option[List[EditorTechniqueCompilationResult]] = None): IOResult[Unit] = ().succeed
```

Comment:
Okay, we need to sync in the status in the `UpdateTechniqueLibrary#update`, there are callbacks that are called, so I think a callback can be added to avoid any change in system API

Reply:
:white_check_mark: added a callback here : https://github.com/Normation/rudder/pull/6068/commits/7787075620aabb1dc4606def7da4fe885425754f